docker-entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+make
+mv wal2json.so /workspace/wal2json-Linux-x86_64.so
```

Comment:
If we add a `ENV DOCKER_ARCH=$DOCKER_ARCH` line into `Dockerfile.builder`, then we can do the following here and make this script architecture independent:
```suggestion
mv wal2json.so /workspace/wal2json-Linux-$DOCKER_ARCH.so
```